cmd/awscollector/Dockerfile

```diff
@@ -10,12 +10,14 @@ RUN apk --update add ca-certificates
 WORKDIR /workspace
 
 ARG TARGETPLATFORM
-RUN echo "Targate platform is $TARGETPLATFORM"
+RUN echo "Target platform is $TARGETPLATFORM"
 
 # copy artifacts
 # always assume binary is created
 COPY build/linux/* /workspace/
-COPY cmd/awscollector/copy_docker_binary.sh  /workspace/cmd/awscollector/copy_docker_binary.sh
+COPY config.yaml /workspace/config/otel-config.yaml
+COPY config/ /workspace/config/
+COPY cmd/awscollector/copy_docker_binary.sh /workspace/cmd/awscollector/copy_docker_binary.sh
```

Comment:
IMO we don't need this line since we don't plan to exec from inside containers anymore.

Reply:
We need this to copy the correct binary into the image.

Reply:
True but we execute the script not inside the container so I don't see any reasons to copy the script to the containers though.

Reply:
The container will not include the script. Only the build container. Only after the FROM scratch will include inside the container

Reply:
True though. But I am still not seeing any reasons why we should copy this file to the ```build container``` if we do not plan to exec inside the ```build container```

Reply:
it is run inside the build container on the next line.

Reply:
Changed the path it copies the script to to `copy_binary.sh` for clarity.

Reply:
Though we could execute script outside the container. However, it does not. Thanks guys for clarifying though.